src/main/resources/default/taglib/t/dot.html.pasta

```diff
@@ -10,6 +10,6 @@
 <span>
 <span class="mb-2 @if(marginRight) { mr-2 } dot-block @class">
     <span class="d-inline-block dot mr-1 bg-sirius-@color">&nbsp;</span>
-    <div><i:render name="body"/></div>
```

Comment:
This has been added on purpose to have a guaranteed (flex row) layout when multiple dots are in a line.